examples/basic_usage/README.md

```diff
@@ -12,8 +12,11 @@ Note that this example may create resources which cost money. Run
 
 | Name | Version |
 |------|---------|
-| terraform | ~> 0.13.0 |
+| terraform | ~> 0.14.0 |
+<<<<<<< HEAD
 | aws | ~> 3.38 |
+=======
+>>>>>>> 0534c57b791b460ddcfb3f7562d99e5bf1c3e7db
```

Comment:
You have some merge turds here.

Reply:
Gah, thanks for catching that!  Fixed with https://github.com/cisagov/freeipa-server-tf-module/pull/53/commits/9077a09cbed3879fb81a113289776b87cf130dff.

Reply:
Heh, well at this point you're at <1% error rate.  That's still pretty darn good in my book!